Observer/OmiseOffsiteDataAssignObserver.php

```diff
@@ -5,18 +5,18 @@
 use Magento\Payment\Observer\AbstractDataAssignObserver;
 use Magento\Quote\Api\Data\PaymentInterface;
 
-class OffsiteInternetbankingDataAssignObserver extends AbstractDataAssignObserver
+class OmiseOffsiteDataAssignObserver extends AbstractDataAssignObserver
 {
     /**
      * @var string
      */
-    const OFFSITE = 'offsite';
+    const SOURCE = 'omise_source_token';
```

Comment:
@jacekstan fyi, Omise is using term of `source` and `token` differently (`source` is for Source API, `token` for Token API. Probably this naming might a bit leading to a confusion?

Reply:
it looks and it is used as token / identifier... what we get and what we send.. but yes... I will apply your suggestions.. thanks..

Reply:
@jacekstan it's ok to not change haha. 
What is it going to be for Credit Card token? `omise_token_token`?

Reply:
Just checked the code, 
I used `omise_card_token` before 🤔 hmm.. maybe I should have named it `omise_token_id` 🤔

Reply:
i will think how to improve naming convention and will improve that after current "sprint"... for now i have updated it to your suggestions.